tensorflow/core/kernels/mkl_conv_grad_filter_ops.cc

```diff
@@ -518,31 +521,33 @@ class MklConv2DCustomBackpropFilterOp
       bias_grad->SetOpMemDesc(bias_grad_dims, memory::format::x);
     }
 
-    // Create convolution backward weights primitive.
-    auto bwd_desc =
-        (biasEnabled && (bias_grad != nullptr))
-            ? convolution_backward_weights::desc(
-                  convolution_direct, input->GetOpMemDesc(),
-                  output->GetOpMemDesc(), bias_grad->GetOpMemDesc(),
-                  outbackprop->GetOpMemDesc(), strides, padding_l, padding_r,
-                  padding)
-            : convolution_backward_weights::desc(
-                  convolution_direct, input->GetOpMemDesc(),
-                  output->GetOpMemDesc(), outbackprop->GetOpMemDesc(), strides,
-                  padding_l, padding_r, padding);
-
-    auto bwd_pd = convolution_backward_weights::primitive_desc(
-        bwd_desc, cpu_engine, conv_fwd_pd);
-
-    // Allocate output tensor.
-    AllocateOutputTensor(context, bwd_pd, bwd_output_dims, bwd_output_format,
-                         output_tensor);
-
-    CHECK_NOTNULL(*output_tensor);
-    // Set buffer handle using allocated output tensor.
-    output->SetUsrMemDataHandle(*output_tensor);
-
     if (biasEnabled && (bias_grad != nullptr)) {
+      // Create convolution backward weights with bias primitive.
+      // Use MKLDNN dilated convolution in case of dilate rate (>0).
+      // MKLDNN dilate rate start from 0.
```

Comment:
What do you mean by "MKLDNN dilate rate start from 0."? 
That comment just seems potentially confusing. Maybe just say "Use dilated convolution in case dilate rates are greater than zero."

Reply:
@rmlarsen thanks for your comment, "MKLDNN dilate rate start from 0." i just want to say for dilate rate is 1 in normal definition ( standard convolution),  in MKLDNN's definition, it is zero.